src/foremast/securitygroup/create_securitygroup.py

```diff
@@ -118,6 +121,40 @@ def _process_rules(self, rules):
         self.log.debug('SG reference rules: %s', non_cidr)
         return non_cidr, cidr
 
+    def add_tags(self, group_id):
+        """
+        Add tags to security group.
```

Comment:
Follow the format of the other doc strings, move this to be right after the `"""` and line between description and args